reg_tests/test_files/SSTTAMSChannelEdge/SSTTAMSChannelEdge_rst.i

```diff
@@ -0,0 +1,228 @@
+
+  - name: solve_scalar
+    type: tpetra
+    method: gmres
+    preconditioner: riluk
+    tolerance: 1e-5
+    max_iterations: 200
+    kspace: 200
+    output_level: 0
+
+  - name: solve_cont
+    type: hypre
+    method: hypre_gmres
+    preconditioner: boomerAMG
+    tolerance: 1e-5
+    max_iterations: 50
+    kspace: 75
+    output_level: 0
+    bamg_coarsen_type: 8
+    bamg_interp_type: 6
+    bamg_cycle_type: 1
+
+realms:
+
+  - name: realm_1
+    mesh: SSTTAMSChannelEdge.rst
+    use_edges: yes 
+    check_for_missing_bcs: yes
+    support_inconsistent_multi_state_restart: yes
+
+    time_step_control:
+     target_courant: 1.0
+     time_step_change_factor: 1.2
+
+    equation_systems:
+      name: theEqSys
+      max_iterations: 4
+
+      solver_system_specification:
+        velocity: solve_scalar
+        turbulent_ke: solve_scalar
+        specific_dissipation_rate: solve_scalar
+        pressure: solve_cont
+        ndtw: solve_cont
+        time_averaged_model_split: solve_cont
+
+      systems:
+        - WallDistance:
+            name: myNDTW
+            max_iterations: 1
+            convergence_tolerance: 1.0e-8
+
+        - LowMachEOM:
+            name: myLowMach
+            max_iterations: 1
+            convergence_tolerance: 1e-8
+
+        - ShearStressTransport:
+            name: mySST
+            max_iterations: 1
+            convergence_tolerance: 1e-8
+
+        - TAMS:
+            name: myTAMS
+            max_iterations: 1
+            convergence_tolerance: 1e-8
+
+    initial_conditions:
+      - constant: ic_1
+        target_name: Unspecified-2-HEX
+        value:
+          pressure: 0
+          velocity: [22.0,0.0,0.0]
+          turbulent_ke: 0.0005
+          specific_dissipation_rate: 0.0005
+          average_velocity: [22.78,0.0,0.0]
+          average_density: 1.0
+          average_tke_resolved: 0.0
+          average_dudx: 0.0
+          k_ratio: 1.0
+          avg_res_adequacy_parameter: 1.0 
+
+    material_properties:
+      target_name: Unspecified-2-HEX
+      specifications:
+        - name: density
+          type: constant
+          value: 1.0
+        - name: viscosity
+          type: constant
+          value: 9.99488e-4 
+
+    boundary_conditions:
+
+    - wall_boundary_condition: bc_bot
+      target_name: bottom
+      wall_user_data:
+        velocity: [0,0,0]
+        turbulent_ke: 0.0
+        use_wall_function: no
+
+    - wall_boundary_condition: bc_top
+      target_name: top
+      wall_user_data:
+        velocity: [0,0,0]
+        turbulent_ke: 0.0
+        use_wall_function: no
+
+    - periodic_boundary_condition: bc_inlet_outlet
+      target_name: [inlet, outlet]
+      periodic_user_data:
+        search_tolerance: 0.001
+
+    - periodic_boundary_condition: bc_front_back
+      target_name: [front, back]
+      periodic_user_data:
+        search_tolerance: 0.001
+
+    solution_options:
+      name: myOptions
+      turbulence_model: sst_tams
```

Comment:
This does use tams....may be this test uses `sst` to converge first and then restart?

Reply:
Yep!